README.md

```diff
@@ -1045,7 +1043,7 @@ Refresh the Code Coverage report in your browser:
 
 ![server-side-test-istanbul-coverage-report](images/server-side-test-istanbul-coverage-report.png)
 
-> _**Boom**_! You know how to run your QUnit-based Unit Tests server-side!
+> _**Boom**_! Now you know how to run your QUnit-based Unit Tests on the server-side!
```

Comment:
We have a confluence of two expressions here now which will need to be changed ➡️  either `server-side!` or `on the server!` 
Both work and `server-side` is more common in tech circles but if you think it's not as accessible to a beginner @BurkHufnagel then `on the server` is cool 👍